charts/osm/values.yaml

```diff
@@ -12,19 +12,19 @@ OpenServiceMesh:
     # -- Container image pull policy for control plane containers
     pullPolicy: IfNotPresent
     # -- Container image tag for control plane images
-    tag: "latest-main"
+    tag: ""
```

Comment:
Would be helpful to add a comment here that the tag must not be set on release branches.